contract_sale_generation/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Pesol (<http://pesol.es>)
+#                Angel Moya <angel.moya@pesol.es>
```

Comment:
This creates ambiguity of your copyright. Either put them on the same line, or rewrite the `Copyright 2017`. My preferred is the latter.